Vagrantfile

```
@@ -266,6 +266,15 @@ Vagrant.configure(2) do |config|
 
   config.vm.provision :shell, :inline => " sudo timedatectl set-timezone America/New_York", run: "once"
 
+
+  # to use the command below, first install the vagrant plugin:
+  #
+  #   vagrant plugin install vagrant-timezone
+  #
+  if Vagrant.has_plugin?("vagrant-timezone")
+    config.timezone.value = "America/New_York"
```

```suggestion
    config.timezone.value = :host
```

Do we want the timezone to be hard-coded NY or match host machine's timezone?

---

`def ensure_plugin_installed(plugin)
  begin
    unless Vagrant.has_plugin?(plugin)
      system "vagrant plugin install #{plugin}"
      print "Waiting for #{plugin} plugin installation to complete."
      until Vagrant.has_plugin?(plugin)
        print "."
        sleep 1
      end
      puts "\n#{plugin} plugin installation completed."
    end
  rescue => e
    raise "Failed to install #{plugin} plugin: #{e.message}"
  end
end

ensure_plugin_installed("vagrant-timezone")`


I was thinking about installing script by default, make it seamless. Not having to mention to each new dev, oh you need to install this plugin

---

Yes...  but Line 267 also has america/new york hard-coded.
(this was already an issue with the vagrantfile before my addition)